Automattic/jetpack
# VideoPress: Create action ADD_VIDEO and invalidate uploaded video count

Follow-up to #26651

#### Changes proposed in this Pull Request:

* Separates the `ADD_VIDEO` action from `SET_VIDEO` to allow for invalidation of the uploaded count when adding a new video

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Go to the VideoPress dashboard
* Add videos and check that the uploaded count is updated as soon as the upload ends
* Check that a new request for the uploaded count is sent
* Delete videos and check that the same happens
* Check that all other uploading behavior is unchanged
